### ISSUE TITLE
Grunt tests involving IBM_DB can't find bindings file

### ISSUE THREAD

**Issue by dexdexdex** [ibmdb/node-ibm_db#171]
We are seeing some super weird behavior on our builds involving IBM_DB. Essentially, a test spec has a module we created that depends on IBM_DB. Recently (in the last couple of days), we noticed that builds would break with this specific error:

```
Running "jasmine_node:unit" (jasmine_node) task
Exception loading: /var/local/jenkins/workspace/common_build_linux_verification/common/test/server/unit/dashboard.spec.js
{ [Error: Could not locate the bindings file. Tried:
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/build/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/build/Debug/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/build/Release/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/out/Debug/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/Debug/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/out/Release/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/Release/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/build/default/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/compiled/0.12.15/linux/x64/odbc_bindings.node]
  tries: 
   [ '/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/build/odbc_bindings.node',
     '/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/build/Debug/odbc_bindings.node',
     '/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/build/Release/odbc_bindings.node',
     '/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/out/Debug/odbc_bindings.node',
     '/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/Debug/odbc_bindings.node',
     '/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/out/Release/odbc_bindings.node',
     '/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/Release/odbc_bindings.node',
     '/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/build/default/odbc_bindings.node',
     '/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/compiled/0.12.15/linux/x64/odbc_bindings.node' ] }
Failed to execute "jasmine.executeSpecsInFolder": Error: Could not locate the bindings file. Tried:
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/build/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/build/Debug/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/build/Release/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/out/Debug/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/Debug/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/out/Release/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/Release/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/build/default/odbc_bindings.node
 → /var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/compiled/0.12.15/linux/x64/odbc_bindings.node
    at bindings (/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/bindings/bindings.js:88:9)
    at Object.<anonymous> (/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/ibm_db/lib/odbc.js:27:31)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/bfdbpool/lib/db2/pool.js:12:10)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/var/local/jenkins/workspace/common_build_linux_verification/common/node_modules/bfdbpool/lib/db2/index.js:10:34)
```

It looks like ibm_db/lib/odbc.js:27:31 is this line here:

```
var odbc = require("bindings")("odbc_bindings")
```

Anyways, I went back and forth to try to fix this. The only thing that actually made things work again was when I reverted to a specific commit of IBM_DB. Like, this package.json WORKS:

```
    "ibm_db": "https://github.com/ibmdb/node-ibm_db.git#76fa625"
```

But the commit immediately afterward fails:

```
    "ibm_db": "https://github.com/ibmdb/node-ibm_db.git#86e7107"
```

Anybody run into this or know what's going on?

**Comment by bimalkjha** [comment]
Error tells you don't have ibm_db/build/Release/odbc_bindings.node file. Means ibm_db have not installed correctly due to some error during installation. Could you please collect that error and share?
The link about two commits looks different, but when I open it points to the same commit. Please share the difference between these two commits which is visible for you. Thanks.

**Comment by bimalkjha** [comment]
@dexdexdex I just pushed a minor change of package.json on github. Please check the latest commit solves your issue or not. Thanks.

**Comment by dexdexdex** [comment]
So I think things actually get fixed when I try running my tests against the latest commit hash that's available. Can we publish this version out into the world? Thanks thanks!

**Comment by bimalkjha** [comment]
@dexdexdex The fix is published as part of ibm_db@1.0.1. Closing the issue. Thanks.
